src/Compilers/Core/Portable/CommandLine/ErrorLogger.cs

```diff
@@ -42,7 +42,7 @@ public ErrorLogger(Stream stream, string toolName, string toolFileVersion, Versi
             _writer = new StreamWriter(stream);
```

Comment:
For this error logger to produce ruleInfo, you'd need to construct/capture rule metadata data instances and then make them retrievable or write on Dispose, which could be cumbersome.